### PR TITLE
Webhook: ability to pass extra arguments to mco

### PR DESCRIPTION
**Pull request by nike38rus**
Continuation of a #362 

This will be useful for finer mco control e.g. to pass `--no-progress` argument to reduce size of access log

Also I made some changes in authentication events logging and fixed identations

### CODE REVIEW

**Comment by rnelson0** [comment]
The change itself looks good, just needs some documentation/examples and an rspec test update.

**Comment by nike38rus** [comment]
@rnelson0 it seems I'm done

File: manifests/webhook/config.pp [hunk 1]
```diff
@@ -44,6 +44,7 @@
   $configfile_symlink         = '/etc/webhook.yaml',
   $enable_mutex_lock          = $r10k::params::webhook_enable_mutex_lock,
   Array $ignore_environments  = $r10k::params::webhook_ignore_environments,
+  $mco_arguments              = $r10k::params::webhook_mco_arguments,
```
Comment (rnelson0) [inline manifests/webhook/config.pp]: This should be a `String`, correct?

Comment (nike38rus) [inline manifests/webhook/config.pp]: Well, if user didn't specified this parameter then default `undef` will be used so I guess it should be `Variant[String, Undef]` ?

Comment (rnelson0) [inline manifests/webhook/config.pp]: I think `Optional[String]` is better.

Comment (nike38rus) [inline manifests/webhook/config.pp]: Indeed!

**Comment by nike38rus** [comment]
@rnelson0 done!

**Comment by rnelson0** [comment]
Thanks for your patience!

**Pull request merged**